android/app/src/main/AndroidManifest.xml

```diff
@@ -23,7 +24,7 @@
 
         <activity
             android:name=".MainActivity"
-            android:label="@string/app_name"
+            android:exported="true"
```

Comment:
Wouldn't this break the display of the application name?

Reply:
No, this attribute does not affect the display of the application name. Build was successfully tested.
- https://developer.android.com/privacy-and-security/risks/android-exported